index.html

```diff
@@ -53,8 +53,23 @@ <h2 class="connect-txt">LET'S CONNECT</h2>
                     <li class="icon"><img src="./images/medium.png" alt="Medium logo"></li>
                 </ul>
             </div>
-        </div>
-    </header>
+        </div> 
+    </header> 
+    <!-- menu section -->
+    <section class="menu">
+        <i class="fa-solid fa-xmark"></i>
```

Comment:
- [ **OPTIONAL** ] I also think it would be better if you add a `cursor pointer` property both on the `Hamburger` and `CloseMenu` buttons to help users immediately understand that they are clickable.